index.html

```diff
@@ -42,6 +42,202 @@ <h4> I’am a Software Developer</h4>
             <li><a href="#"><img src="./images/Disabledm.svg" alt=""></a></li>
         </ul>
     </section>
+    <section class="jobs_container">
+        <h1>My Recent Works</h1>
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![aaar](https://user-images.githubusercontent.com/69638013/159150475-b7f34047-a5b9-4e7d-99f6-cce9e3ae64cb.png)  |  ![rrrrrr](https://user-images.githubusercontent.com/69638013/159150479-e42eeb5a-d669-4ba3-92f6-2b7951584619.png)


- [ ] Kindly center the title and adjust the  border so it looks like the Figma design